### PR TITLE
CaaS P deploy multiple masters (SCRD-4956)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/install_caasp_heat_templates.xml [hunk 1]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+   The installed templates are located in
+   <filename>/usr/share/caasp-openstack-heat-templates</filename>.
+  </para>
+  <para>
+   A working load balancer is needed in your &cloud; deployment. Two
+   load balancer providers are available: Octavia and HAProxy.
+  </para>
+  <para>
+   Verify that load balancing with Octavia or HAProxy is working correctly
+   in your &ostack; installation by creating a load balancer manually and
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: install by* reads better to me, thoughts?

File: xml/install_caasp_heat_templates.xml [hunk 2]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+--provider octavia &lt;<replaceable>SUBNET_ID</replaceable>&gt;
+OR
+&prompt.ardana;neutron lbaas-loadbalancer-create --name test-lb --provider haproxy &lt;<replaceable>SUBNET_ID</replaceable>&gt;</screen>
+  <screen>&prompt.ardana;neutron lbaas-loadbalancer-show &lt;<replaceable>LOAD_BALANCER_ID</replaceable>&gt;
+OR
+&prompt.ardana;openstack loadbalancer show
+&lt;<replaceable>LOAD_BALANCER_ID</replaceable>&gt;</screen>
+  <para>
+   Octavia is the default load balancer provider in &productname; It has some
+   prerequisites for its installation such as the availability of an
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: it appears that there's missing a full stop after &productname;

File: xml/install_caasp_heat_templates.xml [hunk 3]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+   <literal>amphora</literal> image in &o_img;.
+  </para>
+  <para>
+   The steps below can be used to set up a network, subnet, router, security
+   and IPs for a test <literal>lb_net1</literal> network with
+   <literal>lb_subnet1</literal> subnet. If you use this setup for verifying
+   load balancer in the example above, please set <literal>lb_subnet1</literal>
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: for verifying the load balancer*

File: xml/install_caasp_heat_templates.xml [hunk 4]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+   as &lt;<replaceable>SUBNET_ID</replaceable>&gt; in the <command>neutron
+   lbaas-loadbalancer-create</command> commands.
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: command*

File: xml/install_caasp_heat_templates.xml [hunk 5]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+    </substeps>
+   </step>
+  </procedure>
+  <para>
+   <emphasis role="bold">Accessing the Velum &caasp; Dashboard</emphasis>
+  </para>
+  <para>
+   After the stack has been created, the Velum &caasp; dashboard runs on the
+   Admin Node. You can access it using the floating IP address of the Admin
+   Node.
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: Is `Admin Node` (capitlised) the official name?

File: xml/install_caasp_heat_templates.xml [hunk 6]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+    <para>
+     The installed templates are located in
+     <filename>/usr/share/caasp-openstack-heat-templates</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     On the &lcm;, run the <filename>heat-caasp-deploy.yml</filename> playbook
+     and pass parameters for caasp_stack_name, caasp_stack_yaml_file and
+     caasp_stack_env_yaml_file.
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: <literal>caasp_stack_name</literal>, <literal>caasp_stack_yaml_file</literal> and <literal>caasp_stack_env_yaml_file</literal>.

File: xml/install_caasp_heat_templates.xml [hunk 7]
```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+      <para>
+       Username of a valid user that has been set in &o_ident;. Default: admin
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>cpi_password</term>
+     <listitem>
+      <para>
+       Passwordof a valid user that has been set in &o_ident;.
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: Password of *

File: xml/install_caasp_heat_templates.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/install_caasp_heat_templates.xml]: installation works for me

Comment (asettle) [reply xml/install_caasp_heat_templates.xml]: Agreed a few hours later

File: xml/install_caasp_heat_templates.xml [hunk 8]
```diff
@@ -116,7 +116,507 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+   After the stack has been created, the Velum &caasp; dashboard runs on the
+   admin node. You can access it using the floating IP address of the Admin
+   Node.
+  </para>
+  <para>
+   Create an account and follow the steps in the Velum &caasp; dashboard to
+   complete the &caasp; installation.
+  </para>
+  <para>
+   CaaSP Admin Node Install: Screen 1
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: Do these `Admin Nodes` also need to be lowercase too? See all instances below :)

Comment (csymons-suse) [reply xml/install_caasp_heat_templates.xml]: Screen titles left in titlecase...matter of opinion I guess. All other admin node case violations fixed.

Thanks for you diligence.

**Pull request merged**